core/src/main/java/org/everit/json/schema/ToStringVisitor.java

```diff
@@ -227,6 +227,13 @@ private void describePropertyDependencies(Map<String, Set<String>> propertyDepen
         visit(propertyNameSchema);
     }
 
+    @Override void visitDefinitionSchemas(Map<String, Schema> definitionSchemas) {
+        if (!definitionSchemas.isEmpty()) {
+            writer.key("definitions");
```

Comment:
Maybe better to use `$defs` since that is the currently recommended keyword [according to the spec](https://json-schema.org/draft/2020-12/json-schema-core.html#name-schema-re-use-with-defs)?